Userland/Libraries/LibGfx/Vector3.h

```diff
@@ -120,10 +121,13 @@ class Vector3 final {
 
     constexpr void normalize()
     {
-        T inv_length = 1 / length();
-        m_x *= inv_length;
-        m_y *= inv_length;
-        m_z *= inv_length;
+        auto len = length();
+        if (len > 0) {
```

Comment:
This will probably be incredibly unlikely unless it's a mistake on the user's part. Can this be marked as `[[unlikely]]`?

Reply:
Tag @sunverwerth

Reply:
Good point - even if it does not change things in terms of performance, semantically it's better to have an `[[unlikely]]` return out of there.